lib/compat/wordpress-6.0/blocks.php

```diff
@@ -122,3 +122,34 @@ function gutenberg_build_query_vars_from_query_block( $block, $page ) {
 	}
 	return $query;
 }
+
+/**
+ * Workaround for getting discussion settings as block editor settings
+ * so any user can access to them without needing to be an admin.
+ *
+ * @param array $settings Default editor settings.
+ *
+ * @return array Filtered editor settings.
+ */
+function gutenberg_extend_block_editor_settings_with_discussion_settings( $settings ) {
+
+	$settings['discussionSettings'] = array(
```

Comment:
Let's keep this setting experimental for now. We should look for a more general way to expose site options.